### PR TITLE
Changed 1.2 to 1.3 throughout specification

### PR DESCRIPTION
**Pull request by ToddCooper**
Addressed #282 changes.

## 📑 Description

Updated all release 1.2 (and related) release references to 1.3 (and related).  

## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ The SDPi changelog shall not contain the following sections
 
 Each section shall contain a list of action items of the following format: `<brief one-sentence description of what has been done> (#<issue number & URL>).`
 
-## Unreleased
+## [Unreleased]
 
-## [1.3.0] - 2024-03-29
```
Comment (d-gregorczyk) [inline CHANGELOG.md:18]: That does not look right. Why is the 1.3.0 section gone here?

**Comment by d-gregorczyk** [comment]
General remark: There are plenty of "1.2"s throughout the document. We should replace that with a variable as otherwise we have to conduct the same procedure on every release cycle. Tedious...

File: asciidoc/sdpi-supplement-intro.adoc
Comment (JavierEspina) [inline asciidoc/sdpi-supplement-intro.adoc:1]: phrase should read "...updates to release 1.2"

File: asciidoc/sdpi-supplement-intro.adoc [hunk 2]
```diff
@@ -6,11 +6,14 @@
...
-Release 1.2 provides only a few _incremental_ updates to release 1.1; however, it continues the objective of quarterly _minor version_ releases and continues to advance support for prototyping and testing of device-to-device plug-and-trust interoperability.
-It is recognized, though, that this release is incomplete with work continuing for 1.3 and subsequent versions.
+This release of the SDPi 1.3 Standard for Trial Use (HL7 STU) / Trial Implementation (IHE TI) supplement is the first release that integrates resolutions from an HL7 ballot cycle (January 2024). This process implements links from HL7 ballot Jira tickets to IHE DEV.SDPi Github Issues that then enable tracking of the comment resolution implementation in this specification.
+
+Note:  The 1.3.1 release provides a few minor editorial corrections that were overlooked in the first release of 1.3.0.
+
+Release 1.3 provides only a few _incremental_ updates to release 1.3; however, it continues the objective of quarterly _minor version_ releases and continues to advance support for prototyping and testing of device-to-device plug-and-trust interoperability.
```
Comment (JavierEspina) [inline asciidoc/sdpi-supplement-intro.adoc:15]: phrase should read "...updates to release 1.2"

File: asciidoc/sdpi-supplement-intro.adoc [hunk 3]
```diff
@@ -20,7 +23,7 @@ These known limitations include:
 . *Requirements boxes* (e.g., "R1234") have been added especially in TF-2, with some also part of TF-1 and TF-3; this is an initial approach that *_will be significantly expanded in future versions of the supplement_*; an initial approach is provided in <<vol1_clause_sdpi_requirements_modeling_integration>>, with discussion related to how it will be expanded in future versions of the supplement;
 . *_<<term_safe_effective_secure>> Sections_* (see <<vol1_clause_appendix_a_ses_considerations_section_template>>) are included in the specification; however, their use and content will be significantly extended in SDPi 1.x and subsequent versions;
```
Comment (JavierEspina) [inline asciidoc/sdpi-supplement-intro.adoc:24]: Propose to make more future proof by changing "will be significantly extended in SDPi 1.x and subsequent versions" to "will be significantly extended in future versions"

File: asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@
 [%autowidth]
 [cols="1"]
 |===
-| *SDPi Supplement Version Note*: This section is provided as a placeholder for the <<acronym_ics>> table specification that will be added in the subsequent SDPi 1.2 supplement version.
+| *SDPi Supplement Version Note*: This section is provided as a placeholder for the <<acronym_ics>> table specification that will be added in the subsequent SDPi 1.3 supplement version.
```
Comment (JavierEspina) [inline asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc:9]: This should probably be saying "in the subsequent SDPi 1.4 supplement version". However for better maintainability I propose "in a future SDPi supplement version".

**Comment by JavierEspina** [comment]
> General remark: There are plenty of "1.2"s throughout the document. We should replace that with a variable as otherwise we have to conduct the same procedure on every release cycle. Tedious...

I generally agree with the remark. A "current version" label will however not address all occurrences, since we sometimes mention the "current - 1" or "current + 1" version. Maybe we would need those 3 labels? (currentVersion, previousVersion, nextVersion)

**Comment by JavierEspina** [comment]
Shouldn't the version and date shown on the cover page be updated to 1.3.**1** and **today's** date?

File: asciidoc/sdpi-supplement-intro.adoc (hunk 2 above)
Comment (ToddCooper) [reply asciidoc/sdpi-supplement-intro.adoc:15]: Grrrrrr ...

Comment (JavierEspina) [reply asciidoc/sdpi-supplement-intro.adoc:15]: Is that a Grrrr of agreement or disagreement? ;-)

**Pull request merged**